### PR TITLE
HPCC-31197 Make fileservices stats scope consistent with scope format

### PR DESCRIPTION
**Pull request by shamser**
Make fileservices stats scope consistent with scope format

## Type of change:
- [ ] This change is a bug fix (non-breaking change which fixes an issue).
- [x] This change is a new feature (non-breaking change which adds functionality).
- [ ] This change improves the code (refactor or other change that does not change the functionality)
- [ ] This change fixes warnings (the fix does not alter the functionality or the generated code)
- [ ] This change is a breaking change (fix or feature that will cause existing behavior to change).
- [ ] This change alters the query API (existing queries will have to be recompiled)

## Checklist:
- [x] My code follows the code style of this project.
  - [x] My code does not create any new warnings from compiler, build system, or lint.
- [x] The commit message is properly formatted and free of typos.
  - [ ] The commit message title makes sense in a changelog, by itself.
  - [ ] The commit is signed.
- [ ] My change requires a change to the documentation.
  - [ ] I have updated the documentation accordingly, or...
  - [ ] I have created a JIRA ticket to update the documentation.
  - [ ] Any new interfaces or exported functions are appropriately commented.
- [x] I have read the CONTRIBUTORS document.
- [x] The change has been fully tested:
  - [ ] I have added tests to cover my changes.
  - [ ] All new and existing tests passed.
  - [ ] I have checked that this change does not introduce memory leaks.
  - [ ] I have used Valgrind or similar tools to check for potential issues.
- [ ] I have given due consideration to all of the following potential concerns:
  - [ ] Scalability
  - [ ] Performance
  - [ ] Security
  - [ ] Thread-safety
  - [ ] Cloud-compatibility
  - [ ] Premature optimization
  - [ ] Existing deployed queries will not be broken
  - [ ] This change fixes the problem, not just the symptom
  - [ ] The target branch of this pull request is appropriate for such a change.
- [ ] There are no similar instances of the same problem that should be addressed
  - [ ] I have addressed them here
  - [ ] I have raised JIRA issues to address them separately
- [ ] This is a user interface / front-end modification
  - [ ] I have tested my changes in multiple modern browsers
  - [ ] The component(s) render as expected

## Smoketest:
- [ ] Send notifications about my Pull Request position in Smoketest queue.
- [ ] Test my draft Pull Request.

### CODE REVIEW

**Review by jakesmith — DISMISSED** [review]
I think it's okay.
@ghalliday - please sanity check.

**Review by ghalliday — APPROVED** [review]

**Comment by ghalliday** [comment]
@shamser a couple of the roxie tests are failing, and may be related - e.g. despray.ecl

**Review by ghalliday — CHANGES_REQUESTED** [review]
I've eventually tracked it down.
If you change VERBOSE=1 in despray.ecl and rerun it you will see:

Internal Error at ccdcontext.cpp(3844) (in Project 6)

```
    virtual unsigned getWorkflowId() const override
    {
        logctx.CTXLOG("Trying to access WFID from workflow");
        throwUnexpected();
    }
```

File: roxie/ccd/ccdcontext.cpp [hunk 1]
```diff
@@ -3838,12 +3838,6 @@ class CRoxieServerContext : public CRoxieContextBase, implements IRoxieServerCon
         addWuExceptionEx(text, code, 2, MSGAUD_user, "user");
     }
 
-    virtual unsigned getWorkflowId() const override
```
Comment (shamser) [inline roxie/ccd/ccdcontext.cpp:3841]: Use getWorkflowId from base class CRoxieContextBase.

**Review by ghalliday — APPROVED** [review]
@shamser please can you squash

File: roxie/ccd/ccdcontext.cpp (hunk 1 above)
Comment (jakesmith) [reply roxie/ccd/ccdcontext.cpp:3841]: getWorkflowId in that class is:
```
    virtual unsigned getWorkflowId() { return 0; } // this is a virtual which is implemented in IGlobalContext
```

(meaning all fileservice scopes will be "w0:...." )


Looks like this change will avoid the roxie regression test failures, but doesn't seem right.

Unless there's a separate JIRA/PR to address this. If so, there should be some comments in the JIRA/PR to explain.

**Review by jakesmith — COMMENTED** [review]
@shamser - please see comment.

File: roxie/ccd/ccdcontext.cpp (hunk 1 above)
Comment (shamser) [reply roxie/ccd/ccdcontext.cpp:3841]: getWorkflowId is defined as (line 1802)

    virtual unsigned getWorkflowId() const override { return graph->queryWorkflowId(); }
    
Where did you see getWorkflowId() defined to return 0?

**Comment by shamser** [comment]
@jakesmith 

getWorkflowId is defined as (line 1802) as

`virtual unsigned getWorkflowId() const override { return graph->queryWorkflowId(); }`

Where did you see getWorkflowId() defined to return 0?

**Review by jakesmith — APPROVED** [review]
@shamser - looks good.

**Review by ghalliday — APPROVED** [review]

**Pull request merged**